pkg/services/rpcsrv/server_test.go

```diff
@@ -2596,6 +2777,87 @@ func testRPCProtocol(t *testing.T, doRPCCall func(string, string, *testing.T) []
 			checkContract(t, verAcc, invocScript, 146960) // No C# match, but we believe it's OK and it has a specific invocation script overriding anything server-side.
 		})
 	})
+	t.Run("sendrawtransaction", func(t *testing.T) {
+		rpc := `{"jsonrpc": "2.0", "id": 1, "method": "sendrawtransaction", "params": ["%s"]}`
+		t.Run("sendrawtransaction invalid signature", func(t *testing.T) {
```

Comment:
Here and below don't need to specify "sendrawtransaction" in the testcase name, because you have an upper testcase named ""sendrawtransaction" (at L2780).